railties/lib/rails/generators/rails/app/templates/test/system_test_helper.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class ActionSystemTestCase < ActionSystemTest::Base
```

Comment:
Would be nice to make the default `driven_by` explicit here. (As well as rename this `ApplicationSystemTestCase` to infer that this is the base class for THIS application, we are not configuring the top-level framework).